src/app/main/component/user/components/profile/edit-profile/edit-profile.component.scss

```diff
@@ -35,8 +39,11 @@ input:checked ~ .checkmark::after {
   display: block;
 }
 
-.container {
-  margin-left: 135px;
+.edit_prof-container {
+  width: 100%;
+}
```

Comment:
If we will remove these lines of code, will something change?

Reply:
this outer wrapper is serving for taking all the available space from bootstrap container, so we can allocate our content at the very left side of it